src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -861,5 +861,287 @@ public new abstract class Arm64 : AdvSimd.Arm64
         public static unsafe Vector<ulong> LoadVectorUInt32ZeroExtendToUInt64(Vector<ulong> mask, uint* address) => LoadVectorUInt32ZeroExtendToUInt64(mask, address);
 
 
+        ///  UnzipEven : Concatenate even elements from two inputs
+
+        /// <summary>
+        /// svuint8_t svuzp1[_u8](svuint8_t op1, svuint8_t op2)
+        ///   UZP1 Zresult.B, Zop1.B, Zop2.B
+        /// svbool_t svuzp1_b8(svbool_t op1, svbool_t op2)
+        ///   UZP1 Presult.B, Pop1.B, Pop2.B
+        /// </summary>
+        public static unsafe Vector<byte> UnzipEven(Vector<byte> left, Vector<byte> right) => UnzipEven(left, right);
```

Comment:
I just realized that zip and unzip instructions operate on both vector and predicate registers. How do we know which one to invoke? Currently, we will always invoke vector variants? @a74nh ?

If we always going to support just the vector version, then please remove the comment for predicate registers.

Reply:
Right now it'll just be the vector variant. If we're using masks then there will be a lot of conversions and the vector version will be used.

Raised a ticket: https://github.com/dotnet/runtime/issues/101598

In the meantime, is it ok to leave the comment as is?

Reply:
Thanks for opening an issue and yes, ok to keep it as of now.